lib/views/multiline.coffee

```diff
@@ -0,0 +1,21 @@
+class MessageLine extends HTMLElement
```

Comment:
If you would observe the message element, then you would observe that it contains quite a lot of code, that wouldn't be good to fit anywhere else except itself.
I am not saying that You should remove `MultilineElement`. I am however saying that You should remove `LineElement`. 'cause all it does is create a div and set it's text. which could be done in a few lines like
```
el = document.createElement 'div'
e.textContent = Content
el
```